frame/balances/src/lib.rs

```diff
@@ -458,7 +458,7 @@ decl_module! {
 		/// - Contains a limited number of reads and writes.
 		/// # </weight>
 		#[weight = T::DbWeight::get().reads_writes(1, 1) + 100_000_000]
-		fn set_balance(
+		pub fn set_balance(
```

Comment:
Just started reviewing, but you shouldn't need to make this pub. You can instead call `make_free_balance_be` which is part of the currency trait.